### DISCUSSION TITLE
🚀 📚 Feature Request: LTR in Post Navigation Widget

### DISCUSSION THREAD

**Discussion by Tjarkina** [elementor/elementor#21291] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

This serious request is for the RTL developers. The navigation of the Post Navigation Widget is RTL which is not natural for LTR languages. For LTR readers the functionality is now reversed. For them left is past and right is future, left is backwards, and right is forwards. Left is previous and right is next. Left is A and right is B. That's not the case, and that's very confusing for LTR people, so a whole lot  :-)

### Describe the solution you'd like

Maybe it is possible to make something like a switch or a check box to reverse RTL to LTR and vice versa. It's not a bug, but I think the builders are RTL and not LTR. 

### Describe alternatives you've considered

Well, I'm a UX - UI - visual designer. Not a techie that can write a piece of code that can solve this problem, so if someone can write a piece of code that can switch the left and right, the problem is solved too. But better stick to a switch in the widget because half of the world is LTR. 

### Additional context

I hope I did well on my first-ever feature update request, and I really hope you can grant my request. 
@verdiheinz, you asked me to tag you, and @nicholaszein.

**Comment by VerdiH** [comment]
Thanks for taking the time to let our devs know about this @Tjarkina !

**Reply by Tjarkina** [reply]
I hope this will picked up soon @VerdiH
